docs/running-node.md

```diff
@@ -0,0 +1,144 @@
+    * that will require `sudo` for a user
+    * in case non-Ubuntu system these steps will be scipped so you would need to take case about
+
+## Deployment
+
+Pre-requisites:
+1. `dcld` and `dclcli` binaries are located in `/usr/bin`
+2. (for systemd systems) `dcld.service` is available in the current directory
+3. (for validator and observer) `genesis.json` and `persistent_peers.txt` are available in the current directory
+
+### Genesis Validator Node
+
+This part describes how to configure a genesis node - a starting point of any new network.
+
+The following steps automates a set of instructions that you can find in [Running Genesis Node](running-genesis-node.md) document.
+
+1. Choose the chain ID. Every network (for example, test-net, main-net, etc.)
+must have a unique chain ID.
+
+2. run
+
+```bash
+$ run_dcl_node -t genesis -c <chain-id> node0
+```
+
+This command:
+    - configures dclcli to communicate with the genesis node (the local one)
+    - generates a new key entry for a node admin account
+    - generates `genesis.json` file with the following entries:
+        - a genenesis account for the key entry above with `Trustee` and `NodeAdmin` roles
+        - a genesis txn that makes the local node a validator
+    - configures and starts the node
+
+Outputs:
+    - key data (address, public key, mnemonic) as `*.dclkey.json` file in the current directory
+    - genesis file `$HOME/.dcld/config/genesis.json`
+
+## Validator Node
+
+This part describes how to configure a validator node and add it to the existing network.
+
+The following steps automates a set of instructions that you can find in [Running Validator Node](running-validator-node.md) document
+
+1. run
+
+```bash
+$ run_dcl_node -c <chain-id> <node-name>
+```
+
+This command:
+    - configures dclcli to communicate with the genesis node:
+        - by default it will evaluate it from the `persisten_peers.txt` (using its first entry)
+        - you can specify the peer explicitly using `-p/--peer`: `-p <node-id@host:port>` (port is usually considered as `26656` here)
+    - generates a new key entry for a node admin account
+        - by default `<node_name>admin` key name is used
+        - can be configured using `-k/--key-name` option
+    - properly locates `genesis.json`
+    - configures and starts the node
+
+Outputs:
+    - key data (address, public key, mnemonic) as `*.dclkey.json` file in the current directory
+    - node admin key data: `address` and `pubkey`
+    - validator data: `address` and `pubkey`
+
+2. Provide generated node admin key `address` and `pubkey` to any `Trustee`(s). So they may create
+   an account with `NodeAdmin` role. And **wait** until:
+
+   * Account is created
+   * The node completed a catch-up:
+        * `dclcli status --node <node ip>` returns `false` for `catching_up` field
+
+3. Make the node a validator:
+
+```bash
+$ dclcli tx validator add-node \
```

Comment:
What about a passphrase used for account creation? Will the scripts prompt for it?

Reply:
yes